src/main/java/matching_system/MatchManager.java

```diff
@@ -26,10 +28,12 @@ public MatchManager(MatchOutputBoundary matchOutputBoundary, MatchRequestModel m
             userAccountArrayList.add(rm.getUserAccount());
         }
         this.match = new Matcher(user,userAccountArrayList);
+        create(matchRequestModel);
```

Comment:
This method returns MatchResponseModel, right? But the result is not saved? What's the purpose of calling this method here?

Reply:
This function actually also saves the MatchResponseModel to the Presenter, so here I called it to put the MatchResponseModel into the presenter.